server/src/main/java/org/apache/kafka/server/metrics/ClientMetricsConfigs.java

```diff
@@ -131,9 +132,13 @@ private static void validateProperties(Properties properties) {
             }
         });
 
+        Map<String, String> propertiesMap = new HashMap<>(properties.size());
+        properties.forEach((key, value) -> propertiesMap.put((String) key, (String) value));
+        Map<String, Object> parsed = CONFIG.parse(propertiesMap);
```

Comment:
Do we need to create a map or should just write below:

```suggestion
        Map<String, Object> parsed = CONFIG.parse(properties);
```

Reply:
nit: You can also write:

```Map<?, ?> parsed = CONFIG.parse(properties);

Reply:
Thanks for the suggestion. I hadn’t noticed this before.